app/questionnaire_state/item.py

```diff
@@ -0,0 +1,28 @@
+
+
+class Item(object):
+    def __init__(self, id):
+        self.id = id
+        self.sections = []
+        self.children = None
+        self.is_valid = None
+        self.errors = None
+        self.warnings = None
+
+    def update_state(self, user_input):
+        for child in self.children:
+            child.update_state(user_input)
+
+    def get_answer(self, id):
```

Comment:
I wasn't fussed on this method either. I will only return an answer and it returns the first one it encounters. However I take your point about checking that is definitely an answer. I'll have a rethink on this one.